src/base.ts

```diff
@@ -117,7 +117,7 @@ export enum ChainKey {
   EVMT = 'evmt',
   MORT = 'mort',
   FTMT = 'ftmt',
-  CZKT = 'czkt',
+  LNA = 'lna',
```

Comment:
One thing I don't know: Is this a testnet now that will be a mainnet at some point? Do we need Linea and Linea Testnet? 

Will try to find that out

Reply:
Seems like Linea is the network, and the testnet is just Linea Testnet. So `LNAT`?